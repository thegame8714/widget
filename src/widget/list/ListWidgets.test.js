import renderer from 'react-test-renderer'
import React from 'react'
import ListWidget from './ListWidgets'

describe('List of widgets page', () => {
  describe('ListWidget component', () => {
    it('should render the component', () => {
      const listWidgetsComponent = renderer.create(<ListWidget />).toJSON()

      expect(listWidgetsComponent).toMatchSnapshot()
    })
  })
})
