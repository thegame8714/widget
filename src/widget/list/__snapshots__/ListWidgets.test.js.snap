// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List of widgets page ListWidget component should render the component 1`] = `
<h1
  className="sc-bdVaJa ghfIaL"
>
  List of widgets
</h1>
`;
